py/strato/common/log/configurelogging.py

```diff
@@ -114,16 +116,43 @@ def _configureOutputToFile(logger, logName):
     handler = logging.FileHandler(filename=logFilename(logName))
     atexit.register(handler.close)
     handler.setFormatter(machinereadableformatter.MachineReadableFormatter())
-    handler.set_name("file")
+    handlerName = "file_%s" % logName if logName != _name else "file"
+    handler.set_name(handlerName)
     handler.setLevel(logging.DEBUG)
     logger.addHandler(handler)
     logger.findCaller = _findCaller
 
-
-def _configureLogLevels():
+def _configureLogLevels(name):
     if config.LOG_CONFIGURATION is not None:
         with open(config.LOG_CONFIGURATION, 'rt') as f:
             dictConfig = json.load(f)
     else:
         dictConfig = config.DEFAULT_LOG_CONFIGURATION
+    if os.path.exists(config.LOGS_CONFIGURATION_OVERRIDE_FILE):
+        with open(config.LOGS_CONFIGURATION_OVERRIDE_FILE, 'rt') as f:
+            overrides = json.load(f)
+        default_overrides = overrides.get('default_log_overrides', {})
+        dictConfig.update(default_overrides)
+        dictConfig.update(overrides.get(name, {}))
     logging.config.dictConfig(dictConfig)
+
+def _reloadLoggingConfiguration(signal, stackFrame):
+    global _name
+    if _name is not None:
+        _configureLogLevels(_name)
+
+def _getMultipleFuncsHandler(funcs):
+    def _multipleFuncsHandler(signalNumber, stackFrame):
+        for func in funcs:
+            func(signalNumber, stackFrame)
+    return _multipleFuncsHandler
+
+def _configureLoggingSignalHandlers():
+    currentHandler = signal.getsignal(config.UPDATE_LOGGING_CONFIGURATION_SIGNAL)
```

Comment:
Thank you for your comment @sasha-stratoscale. I believe https://github.com/Stratoscale/pycommonlog/pull/3, provides a solution for your concerns.
A user will be able to not register the signal handler, and call it from his conf reload.